ostd/src/mm/frame/mod.rs

```diff
@@ -57,6 +57,13 @@ use crate::mm::{Paddr, PagingConsts, Vaddr};
 
 static MAX_PADDR: AtomicUsize = AtomicUsize::new(0);
 
+/// Returns the maximum physical address that is tracked by frame metadata.
+pub fn max_paddr() -> Paddr {
```

Comment:
Also, restrict visibility a bit if it does not look useful for outer crates/modules? We can relax later if there are some interesting use cases.